verifier/src/lib.rs

```diff
@@ -83,6 +83,14 @@ pub fn verify(
             let opts = AcceptableOptions::OptionSet(vec![
                 ProvingOptions::RECURSIVE_96_BITS,
                 ProvingOptions::RECURSIVE_128_BITS,
+                ProvingOptions::RECURSIVE_96_BITS.with_partitions(1, 8),
+                ProvingOptions::RECURSIVE_128_BITS.with_partitions(1, 8),
+                ProvingOptions::RECURSIVE_96_BITS.with_partitions(2, 8),
+                ProvingOptions::RECURSIVE_128_BITS.with_partitions(2, 8),
+                ProvingOptions::RECURSIVE_96_BITS.with_partitions(3, 8),
+                ProvingOptions::RECURSIVE_128_BITS.with_partitions(3, 8),
+                ProvingOptions::RECURSIVE_96_BITS.with_partitions(4, 8),
+                ProvingOptions::RECURSIVE_128_BITS.with_partitions(4, 8),
```

Comment:
Please double check if these make sense to you.

Reply:
I removed the option with 3 partitions.

My intuition is that it's better to add it if somebody needs it (and can ensure that it works), rather than have an option that may or may not be used and therefore may or may not work in a future release.